src/views/SystemIntake/index.tsx

```diff
@@ -52,6 +58,16 @@ export const SystemIntake = () => {
   const isSubmitting = useSelector((state: AppState) => state.action.isPosting);
   const prevIsSubmitting = usePrevious(isSubmitting);
 
+  const { loading, data } = useQuery<GetSystemIntake, GetSystemIntakeVariables>(
+    GetSystemIntakeQuery,
```

Comment:
@jim @sarboc I was thinking we'd have separate queries for each page. One of the big upsides of GraphQL is that we  don't need to "over-fetch" data. I'm curious what ya'll think of having queries for the data required of each page.

Reply:
I think that sounds like a good idea, but maybe some fine tuning we can do after the larger cutover? That means we would need to support multiple queries on the front end, and also think about how much we want the index component to do here, versus the individual pages of the form. All totally worth thinking about, but with so many moving parts, I wonder if we should let the dust settle a bit before we start doing too much work on the front end.